src/api/app.js

```diff
@@ -12,9 +13,35 @@ app.get('/build/bundle.js', (req, res) => res.sendFile(path.resolve('build/bundl
 app.get('/login', (req, res) => {
   const params = {
     client_id: process.env.GITHUB_CLIENT_ID,
-    redirect_uri: process.env.BASE_URL + '/welcome'
+    redirect_uri: process.env.BASE_URL + '/welcome',
+    scope: 'user repo read:org'
   };
   res.redirect('https://github.com/login/oauth/authorize?' + qs.stringify(params));
 });
 
+app.get('/welcome', (req, res) => {
+  const payload = {
+    client_id: process.env.GITHUB_CLIENT_ID, // given to you when you register your app with github
+    client_secret: process.env.GITHUB_CLIENT_SECRET, // given to you when you register your app with github
+    code: req.query.code // temporary code from github in query of redirect
+  };
+  const url = 'https://github.com/login/oauth/access_token';
+  const headers = {
+    Accept: 'application/json',
+    'User-Agent': 'Gitodoro'
+  };
+  const options = {
+    url,
+    form: payload,
+    headers
+  };
+  request.post(options, (err, response, body) => {
+    if (err) console.log(err);
+    else {
+      var token = JSON.parse(body).access_token;
```

Comment:
I'm surprised the linter didn't pick up on this, but can you change the var to const